care/facility/api/serializers/patient.py

```diff
@@ -17,18 +17,31 @@
 from config.serializers import ChoiceField
 
 
-class PatientSerializer(serializers.ModelSerializer):
+class PatientListSerializer(serializers.ModelSerializer):
```

Comment:
There will be 3 Patient serializers now.
PatientList, PatientDetail and Patient

Try reusing the patient detail